Packs/ProofpointEmailSecurity/Integrations/ProofpointEmailSecurityEventCollector/ProofpointEmailSecurityEventCollector.yml

```
@@ -0,0 +1,46 @@
+commonfields:
+  id: Proofpoint Email Security Event Collector
+  version: -1
+name: Proofpoint Email Security Event Collector
+display: Proofpoint Email Security Event Collector
+category: Analytics & SIEM
+description: Collects events for Proofpoint using the streaming API.
+configuration:
+- display: Server Host
+  name: host
+  type: 0
+  defaultvalue: logstream.proofpoint.com:443
+  required: true
+- display: Cluster ID
+  name: cluster_id
+  type: 0
+  required: true
+- displaypassword: API key
+  name: api_key
+  hiddenusername: true
+  type: 9
+  required: true
+- display: Fetch interval in seconds
+  name: fetch_interval
+  type: 0
+  defaultvalue: 60
+  required: true
+- display: Use system proxy settings
+  name: proxy
+  type: 8
+  required: false
+script:
+  script: ""
+  type: python
+  commands:
+    - description: This command is not implemented. Enable long running to collect events.
+      name: proofpoint-get-events
+
```

I think we can remove the command entirely here. 
```suggestion
  commands: []
```